Testing/functional/e2e/cypress/integration/timeline/errors.js

```diff
@@ -0,0 +1,31 @@
+const { AuthMethod } = require("../../support/constants")
+
+describe('Banner Error', () => {
+    before(() => {
+        cy.intercept("GET", "/v1/api/Note/*", (req) => {
+            req.reply((res) => {
+                res.body.resultStatus = 0;
+                res.body['resultError'] = {
+                    resultMessage: "Error ABC",
+                    errorCode: "Error Code",
```

Comment:
The translator wont be able to put this in the correct format

Reply:
ahh right, let me change to test the translation as well

Reply:
updated!